src/database.ts

```diff
@@ -691,6 +836,11 @@ class Database extends ServiceObject {
       callback!
     );
   }
+
+  getSchema(): Promise<[string[], r.Response]>;
```

Comment:
```suggestion
  getSchema(): Promise<[string[]]>;
```
I do not think r.Response is returned here. 
Check please

Reply:
I think it is required as per document below